src/main/java/ejercicio/banco/repository/CsvAccountRepository.java

```diff
@@ -32,24 +32,43 @@ public CsvAccountRepository() {
         // Creates an user ID starting at 1, auto-incrementing every new user
         int userId = 1;
 
-        List<String> in = reader.read(filename);
+        List<String> in = null;
+
+
+        //Tries to read file. If file not found, returns an empty account list
+
+        try {
+            in = reader.read(filename);
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+            return accounts;
+        }
 
         for (int x = 0; x < in.size(); x++) {
             String line = in.get(x);
             System.out.println(line);
 
             // Assuming that the info is separated by "; " ,splits it
-            String[] parts = line.split(";");
-            String userName = parts[0];
-            float money = Float.valueOf(parts[1]);
-            String iban = parts[2];
-
-            //Prints user into console
-            System.out.println("User " + userId + " named " + userName + " has been processed");
+                String[] parts = line.split(";");
+            String userName;
+            float money;
+            String iban;
+
+                //tries to read account
+                try {
+                    userName = parts[0];
+                    money = Float.valueOf(parts[1]);
+                    iban = parts[2];
+
+                    //Prints user into console
+                    System.out.println("User " + userId + " named " + userName + " has been processed");
+
+                    // Creates a new account and adds it to the list
+                    Account acc = new Account(userId, userName, money, iban);
+                    accounts.add(acc);
+                }
+                catch(Exception e){}
```

Comment:
One question, why do you need to catch here the exception? As far as I can see, here no exception can be thrown, am I right?